engine/src/main/java/org/hibernate/search/query/dsl/impl/ConnectedMultiFieldsRangeQueryBuilder.java

```diff
@@ -116,12 +116,16 @@ private static Query createKeywordRangeQuery(String fieldName, RangeQueryContext
 		final Object fromObject = rangeContext.getFrom();
 		final Object toObject = rangeContext.getTo();
 		final DocumentBuilderIndexedEntity documentBuilder = Helper.getDocumentBuilder( queryContext );
-		final String fromString = fieldContext.objectToString( documentBuilder, fromObject, conversionContext );
+		final String fromString = fromObject == null ?
+				null :
+				fieldContext.objectToString( documentBuilder, fromObject, conversionContext );
 		final String lowerTerm = fromString == null ?
 				null :
 				Helper.getAnalyzedTerm( fieldName, fromString, "from", queryAnalyzer, fieldContext );
 
-		final String toString = fieldContext.objectToString( documentBuilder, toObject, conversionContext );
+		final String toString = toObject == null ?
```

Comment:
So I would add another jira for banning below(null) and above(null) if you agree.